server/app.js

```diff
@@ -1,6 +1,22 @@
 const express = require("express");
 const path = require("path");
 const app = express();
+const { Octokit } = require("@octokit/core");
+const dotenv = require("dotenv");
+require("dotenv").config();
+
+const {
+  paginateRest,
+  composePaginateRest,
+} = require("@octokit/plugin-paginate-rest");
+
+const MyOctokit = Octokit.plugin(paginateRest);
+
+const octokit = new MyOctokit({
```

Comment:
iirc, the github auth token is only necessary if we're using Octokit's graphql apis, i think if we're just using the plain octokit core functionality then an auth token is unnecessary! 

We probably also don't want to use a github auth token as well since it'll make it hairy for deployment in the future having to add in a process environment variable and i think it would be unnecessary for our project!

Reply:
Thank you! I actually tried not using an auth token. But without it, no repo information would be shown.